jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -35,84 +35,88 @@ set -e
 : ${CONNECTION:?"The CONNECTION environment variable must be set."}
 : ${SITESPEED_BROWSER:?"The SITESPEED_BROWSER environment variable must be set."}
 
-if [ $GET_URLS_FROM_FILE == "true" ] ; then
-	msg="The SITESPEED_URL_FILE environment variable must be set to the file containing the URLs to test."
-	: ${SITESPEED_URL_FILE:?$msg}
+# Test for the existence of TEST_URL_FILE
+if [ ! -z ${TEST_URL_FILE+x} ] ; then
```

Comment:
oic. That totally makes sense. I think that since it's pretty tricky it would be good to add a comment that says something like:
Test that the TEST_URL_FILE variable is Set by substituting its value with "x" if it is either Null or Not Null, and then checking if the result is Null. This will only be the case if the original value was Unset.